code-reviews.md

```diff
@@ -1,36 +1,137 @@
+
+if either of these two files are not present, go straight to the issues section of the repo and check if someone else has already raised this as an issue. If no issue exists highlighting the lack of documentation, raise one.
+
+### 3. Quick Check or Detailed Review?
+
+Gauge what level of review is required.  
+Are you doing a detailed review prior to delivering a sprint to a client or deploying a major version to many 'users'?
+Or is this an small/incremental change small enough for you to understand everything that has changed in 20mins (or less) of reading?
+
+Do you have access to the "user" stories / acceptance tests, to know if the code has solved the problem as it was originally identified?
+
+### 4. Large Project? > Focus on the files that are changing the most
+
+If you are new to the project, it can be overwhelming and you might not know where to start looking in the codebase. If the project organisation is not *obvious* after 5 minutes of inspection, call for help.
+If the code author(s) are available, try to pick their brains and request a walk-through (either in person or remote. e.g: google hangout)
+
+### 5. Raising issues
+
+The main branch of a repository should be "*stand alone*".
+By this we mean it should be self-explanatory (without further clarification from the original creators).
+
+Raise issues if you spot any gaps in clarity in the code or documentation.
+
+#### Etiquette
+
+As the person reviewing code, always phrase your issues in an open-minded way (*avoid "this is bad code" type issues, they are useless*) use questions and *seek to understand*.
+
+If you (*think* you) know a "better way" of solving a problem, phrase the issue as a question: e.g:  
+"*Would you consider using `this` here instead of `that` because of xyz reason...?*"
+
+If you are referring to a section of code, provide a full link to the line(s) including the branch/commit id. e.g: https://github.com/foundersandcoders/beekeeper/blob/e92ef2d3625ea5e5f7cf29e7daa11c86fa3741bd/test/auth.test.js#L20
+
+Tip: if you want to keep links short, use http://git.io/
 
-###Why review code? 
-Learning to read code is as important as learning to write it. So, to assist your own learning and, incidentally, to help others.   
-###How to review code 
-By raising issues in a GitHub repo.     
 If you are reviewing a Pull Request add line-comments: https://help.github.com/articles/commenting-on-the-diff-of-a-pull-request/ so that your feedback is contextual.  
 
-###GitHub 
-+ Clone the repo; 
-+ Follow the instructions in the README.  
+#### Guide: Ask yourself (*the code author*) the following *Questions*
 
-###Raise issues 
-+ If the code doesn't run when using `npm i && npm test`; 
-+ If the installation instructions are incomplete; 
-+ If the instructions for running tests are incomplete; 
+
++ Does this pass the "everyone on the project gets abducted by aliens" test?
++ If the code doesn't run when using `npm i && npm test` (*for node.js based projects...*);
```

Comment:
Yeah, agree, it makes sense for ***all*** these bullet points to be phrased as questions rather than if statements. I will make time today to update them. :+1: